### PR TITLE
Addition of `Return` and `Space` buttons

### PR DESCRIPTION
**Pull request by scipianne**

### CODE REVIEW

File: ChemKeyboard/Extensions/UIInputViewController+Extensions.swift [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+public extension UIInputViewController {
+    
+    public func returnKeyString() -> String {
+        if let type = textDocumentProxy.returnKeyType {
+            switch type {
+            case .`default`:
+                return "RETURN"
+            case .go:
+                return "GO"
```
Comment (ogrenich) [inline ChemKeyboard/Extensions/UIInputViewController+Extensions.swift]: @scipianne Can we return something like `type.rawValue.capitalized` here for all cases except `default`?

File: ChemKeyboard/Extensions/UIInputViewController+Extensions.swift [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+//
+//  Created by Maria Dagaeva on 16.11.17.
+//  Copyright © 2017 Chemistry Open Intelligence Pte. Ltd. All rights reserved.
+//
+
+import UIKit
+
+public extension UIInputViewController {
+    
+    public func returnKeyString() -> String {
```
Comment (ogrenich) [inline ChemKeyboard/Extensions/UIInputViewController+Extensions.swift]: @scipianne  Maybe better will be `var` (not `func`) here for clear code with calls of this extension in other classes. – What do you think about it?

File: ChemKeyboard/Extensions/UIInputViewController+Extensions.swift (hunk 1 above)
Comment (scipianne) [reply ChemKeyboard/Extensions/UIInputViewController+Extensions.swift]: Nope, rawValue is Int 😞

File: ChemKeyboard/Extensions/UIInputViewController+Extensions.swift (hunk 2 above)
Comment (scipianne) [reply ChemKeyboard/Extensions/UIInputViewController+Extensions.swift]: sure, that would be better

**Pull request merged**